packages/components/src/components/SquiggleErrorAlert.tsx

```diff
@@ -1,43 +1,77 @@
-import { SqError, SqFrame } from "@quri/squiggle-lang";
-import React from "react";
+import { FC, PropsWithChildren } from "react";
+
+import {
+  SqCompileError,
+  SqError,
+  SqFrame,
+  SqRuntimeError,
+} from "@quri/squiggle-lang";
+
 import { ErrorAlert } from "./Alert.js";
+import { useViewerContext } from "./SquiggleViewer/ViewerProvider.js";
 
 type Props = {
   error: SqError;
 };
 
-const StackTraceFrame: React.FC<{ frame: SqFrame }> = ({ frame }) => {
+const LocationLine: FC<{
+  location: NonNullable<ReturnType<SqFrame["location"]>>;
+}> = ({ location }) => {
+  const { editor } = useViewerContext();
+
+  const findInEditor = () => {
+    editor?.scrollTo(location.start.offset);
+  };
+
+  return (
+    <span className="cursor-pointer hover:text-red-900" onClick={findInEditor}>
```

Comment:
I'd probably add an underline, to make it clear this is clickable. Can do later though.

Also, I'm not sure if this complexity is worth it at this point.

Reply:
Stacktraces can be long, so default underline styles feel a bit noisy:
<img width="239" alt="Captura de pantalla 2023-06-25 a la(s) 17 30 56" src="https://github.com/quantified-uncertainty/squiggle/assets/89368/08fbd088-b1bd-4ae6-b178-d7e3c9ff7f52">

There's not much added complexity here, since we use the same handle for "find in editor" interaction. (Other changes in this file are because of `SqRuntimeError | SqCompileError` refactoring, "clickable" feature is just 4 lines of code).

I agree centering would be good.